charts/artifact-caching-proxy/templates/statefulset.yaml

```diff
@@ -27,9 +32,13 @@ spec:
               ]
             }
           }
-        {{- if .Values.logCollection.enabled }}
-        # log collection with Datadog
-        ad.datadoghq.com/{{ .Chart.Name }}.logs: {{ .Values.logCollection.config | squote }}
+        {{- end }}
+        {{- if .Values.datadog.logCollection.enabled }}
+        # Nginx Log collection with Datadog:
+        # - https://docs.datadoghq.com/integrations/nginx/?tab=kubernetes#log-collection
+        # - https://docs.datadoghq.com/containers/kubernetes/log/?tab=kubernetes#configuration
+        #   1. With 'ad.datadoghq.com/xxx.logs' , the 'xxx' must be the name of the pod's container name
+        ad.datadoghq.com/{{ .Chart.Name }}.logs: '[{"source":"nginx","service":"{{ .Chart.Name }}","tag": "env:prod"}]'
```

Comment:
```suggestion
        ad.datadoghq.com/{{ .Chart.Name }}.logs: '[{"source":"nginx","service":"{{ .Chart.Name }}"}]'
```

Reply:
Looks good: tag specification should be configurable through value as you mentioned.